### PR TITLE
Switching to GHA

### PR DESCRIPTION
**Pull request by cderv**
Same as https://github.com/yihui/servr/pull/50

I just don't have the right to merge here. You'll need to do it @yihui

### CODE REVIEW

**Comment by cderv** [comment]
And it seems it won't trigger Actions as I don't have push access. SO I can't check that this works. 

@yihui I'll debut if necessary after merging. This is just the template action with slight modifications

**Review by yihui — APPROVED** [review]
Thanks!

**Pull request merged**
